Resources/public/js/views/fields/ez-time-editview.js

```diff
@@ -104,13 +172,48 @@ YUI.add('ez-time-editview', function (Y) {
          * @return {Number}
          */
         _getFieldValue: function () {
-            var valueOfInput = this._getInputNode().get('valueAsNumber');
+            var valueOfInput,
+                time,
+                hoursTimestamp,
+                minutesTimestamp,
+                secondsTimestamp = 0;
 
-            if (valueOfInput) {
-                return valueOfInput/1000;
+            if (this.get('supportsTimeInput')) {
+                valueOfInput = this._getInputNode().get('valueAsNumber');
+                if (valueOfInput) {
+                    return valueOfInput/1000;
+                }
+                return null;
+            } else {
+
+                time = this._getInputNode().get('value').split(":");
+                hoursTimestamp = Y.Number.parse(time[0])*3600;
+                minutesTimestamp = Y.Number.parse(time[1])*60;
+                if ( time.length > 2 ) {
+                    secondsTimestamp = Y.Number.parse(time[2]);
+                }
+                time = hoursTimestamp + minutesTimestamp + secondsTimestamp;
+                if (time !== 0){
```

Comment:
bug here :) 0 is midnight so this method can return 0
And in any case it's useless to transform a number of seconds in ... a number of seconds :p